apps/web/components/question/Question.test.tsx

```diff
@@ -117,7 +117,10 @@ describe("Question", () => {
 
     // add questionCodeReference if it does not exist in questionData
     if (!questionInfo.questionData.questionCodeReference) {
-      questionInfo.questionData.questionCodeReference = "code reference";
+      questionInfo.questionData.questionCodeReference = {
+        displayString: "code reference",
+        codeNumber: "1",
```

Comment:
Should we check the value of the code reference in the test instead of just that it's there?

Reply:
Good call. Added that expect!